init.lua

```diff
@@ -76,9 +79,9 @@ elseif has_mcl then
 		'listring[current_player;main]'..
 		'listring[current_name;src]'..
 		'listring[current_player;main]'
-	groups = {axey=2, handy=2, flammable=-1, container = 4}
+	groups = {axey=2, handy=1, flammable=-1, container = 4}
```

Comment:
Why is this change required?

Reply:
"Handy = 2" doesn't exist in mcl games. "Handy = 1" is what's required to allow the node to drop when dug by hand; otherwise it's destroyed. Increasing the hardness value is how the dig time is increased.

Reply:
Ah sorry, i thought the `handy` group does exist in mtg too. Then this is fine-